soul-metrics/soul-metrics-spi/src/test/java/org/dromara/soul/metrics/api/SummaryMetricsTrackerTest.java

```diff
@@ -20,22 +20,24 @@
 import org.dromara.soul.metrics.enums.MetricsTypeEnum;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
+import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 
 /**
  * Test for SummaryMetricsTracker default method.
  *
  * @author dengliming
+ * @author Young Bean
```

Comment:
in pr #948 all java doc is deleted, I don't what is the standard in test case now